File: Pizza-Delivery-System/Pizza-Delivery-System/Models/Pizza.cs
namespace Pizza_Delivery_System.Models
{
    public class Pizza : BaseModel
    {
        public string Image { get; set; }
        public decimal Totalprice { get; set; }
        public string? Description { get; set; }
        public List<PizzaItem> Sauce { get; set; }
        public List<PizzaItem> Toppings { get; set; }
        public List<PizzaItem> Cheese { get; set; }
        public List<PizzaItem> Size { get; set; }
        public Pizza()
        {
            Image = string.Empty;
            Sauce = new List<PizzaItem>();
              Toppings = new List<PizzaItem>();
            Cheese = new List<PizzaItem>();
            Size = new List<PizzaItem>();

        }

    }
}

File: Pizza-Delivery-System/Client/src/app/Models/Pizza.ts
import { BaseModel } from "./BaseModel";
import { PizzaItem } from "./PizzaItem";


export class Pizza extends BaseModel {
  
  Image:string;
  Totalprice:number;
  Description:string;
  Sauce:Array<PizzaItem>;
  Toppings:Array<PizzaItem>;
  Cheese:Array<PizzaItem>;
  Size:Array<PizzaItem>;
  constructor() {
    super();
   ;
   this.Image="";
   this.Description="";
   this.Totalprice=0;
   this.Sauce=new Array<PizzaItem>();
   this.Toppings=new Array<PizzaItem>();
   this.Cheese=new Array<PizzaItem>();
   this.Size=new Array<PizzaItem>();
  }

}